src/Components/AddTask/AddTask.tsx

```diff
@@ -7,12 +7,11 @@ import { AuthContext } from '../../Context/AuthContext'
 export default function AddProject() {
  const navigate= useNavigate()
 const {register,handleSubmit,formState:{errors}}=useForm()
-let{baseUrl,reqHeaders}:any=useContext(AuthContext)
+const{baseUrl,reqHeaders}:any=useContext(AuthContext)
 const AddProject =(data)=>{
-axios.post(`${baseUrl}/Task`,data,{headers:reqHeaders}).then((response)=>{
-  
+axios.post(`${baseUrl}/Task`,data,
+{headers:reqHeaders}).then((response)=>{
   navigate(-1)
-  
 }).catch((error)=>{
   console.log(error);
```

Comment:
consider removing logs before merging to the main branch, to avoid revealing sensitive data to users.